docs/documentation/server_admin/topics/authentication/flows.adoc

```diff
@@ -403,6 +403,25 @@ Note when the login request initiates a request with the `claims` parameter requ
 one of the specified levels. If it is not able to return one of the specified levels (For example if the requested level is unknown or bigger than configured conditions
 in the authentication flow), then {project_name} will throw an error.
 
+[[_registration-rc-client-flows]]
+==== Registration or Reset credentials requested by client
+
+Usually when user is redirected to the {project_name} from client application, the `browser` flow is triggered. This flow may allow user to <<con-user-registration_{context}, register>> in case
```

Comment:
```suggestion
Usually when the user is redirected to the {project_name} from client application, the `browser` flow is triggered. This flow may allow the user to <<con-user-registration_{context}, register>> in case
```